### PR TITLE
Fix broken OverlayTrigger tooltips

### PR DESCRIPTION
**Pull request by skobyda**
If you put OverlayTrigger over a DISABLED button, then overlay tooltip
won't show up. The Reason is probably that a disabled button doesn't
have a pointer event, therefore OverlayTrigger doesn't know when
to showa  tooltip.
Fix this by setting pointerEvents to "none", which will "trigger event
listeners on parent element".

### CODE REVIEW

**Comment by skobyda** [comment]
Fixes tooltip like this one, which would not show up:
Screenshot from 2019-11-18 15-21-35

**Comment by martinpitt** [comment]
That is a really weird hack. We don't need this anywhere else, e. g. in pkg/kdump/kdump-view.jsx . Is that a bug in PF4 button or something, that it *does* swallow the event even when disabled? Normally they should propagate to the non-disabled parent (the OverlayTrigger).

File: pkg/machines/components/create-vm-dialog/createVmDialog.jsx [hunk 1]
```diff
@@ -891,14 +891,20 @@ export class CreateVmAction extends React.Component {
...
         );
         if (!this.state.virtInstallAvailable)
             createButton = (
-                <OverlayTrigger overlay={ <Tooltip id='virt-install-not-available-tooltip'>{ _("virt-install package needs to be installed on the system in order to create new VMs") }</Tooltip> } placement='top'>
-                    {createButton}
+                <OverlayTrigger overlay={
+                    <Tooltip id='virt-install-not-available-tooltip'>
+                        {_("virt-install package needs to be installed on the system in order to create new VMs")}
+                    </Tooltip>} placement='top'>
+                    <span>
```
Comment (martinpitt) [inline pkg/machines/components/create-vm-dialog/createVmDialog.jsx]: Why does it require that extra span?

**Comment by skobyda** [comment]
@martinpitt Apparently it's a bootstrap bug (https://github.com/twbs/bootstrap/issues/16088) which then propagates into Patternfly. There is a lot of discussion how to work around that bug, this is one solution.
The reason why k-dump doesn't need it for example is that we only set the button do be disabled throught "disabled" css class, without actually setting disabled attributes.

**Comment by skobyda** [comment]
Alternatively, we could do it the way k-dump page does that. Instead of setting disable attribute, which triggers the bootstrap bud, we could only set the class to disabled to make the button look disabled, turn of onClick attributes and then the button would act like disabled and have working OverlayTrigger.

**Pull request merged**